docs/openapi-issuer.yaml

```diff
@@ -0,0 +1,570 @@
+openapi: 3.0.3
+info:
+  title: Firma con IO - Issuer API
+  version: 1.0.0
+servers:
+  - url: https://api.io.pagopa.it/api/v1/sign
+    description: production
+security:
+  - SubscriptionKey: []
+paths:
+  /dossiers:
+    post:
+      operationId: createDossier
+      tags:
+        - Dossier
+      summary: Create a Dossier
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateDossierBody"
+      responses:
+        "201":
+          description: The Dossier detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DossierDetailView"
+        "400":
```

Comment:
All API that pass though the API management must handle: 400, 401, 403, 429.

see https://github.com/pagopa/io-functions-services/blob/master/openapi/index.yaml#L296